ILRepack.IntegrationTests/Peverify/PeverifyHelper.cs

```diff
@@ -0,0 +1,49 @@
+﻿using GitHub.Helpers;
+using Microsoft.Win32;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using System.Reactive.Subjects;
+using System.Text.RegularExpressions;
+
+namespace ILRepack.IntegrationTests.Peverify
+{
+    public static class PeverifyHelper
+    {
+        static Regex Success = new Regex(@"All Classes and Methods in .* Verified");
+        static Regex Failure = new Regex(@"\d+ Error\(s\) Verifying .*");
+        public static IObservable<string> Peverify(string workingDirectory, params string[] args)
+        {
+            // TODO better path finding ?
```

Comment:
I hesitated to do that (hence the bottom unused helper method), but also realized that they started publishing MSBuild (fat'ish, tools'ish) nuget package, where I could use MSBuild.Utilities ToolsWhateverLocationHelper.OneOfManyGetSdkPathMethod.

I'm still hesitating, hence the TODO to move foward on the bug fixing, and I was sure you would help me get out of my hesitation :smile: